code/modules/reagents/reagent_containers.dm

```diff
@@ -22,10 +22,15 @@
 	if(!usr.Adjacent(src) || !ishuman(usr) || HAS_TRAIT(usr, TRAIT_HANDS_BLOCKED))
 		return
 
-	var/default = null
-	if(amount_per_transfer_from_this in possible_transfer_amounts)
-		default = amount_per_transfer_from_this
-	var/N = input("Amount per transfer from this:", "[src]", default) as null|anything in possible_transfer_amounts
+	var/picked_amount
+	if(length(possible_transfer_amounts) == 1) // If theres only one option, pick it for them.
+		picked_amount = pick(possible_transfer_amounts)
```

Comment:
why bother setting the picked_amount if it's always going to be the same value? Also why not just early return here?

Reply:
The idea behind it was that if you somehow got it on say "50" units but then that somehow got removed and the only option was 20 units, you would still be able to change it to 20 units.

Reply:
is there a current case in the code base where this would happen?

Reply:
No but it can be done really easily with var-editting, and it stops stuff with it being stuck on stuff like that.

Reply:
Mirroring from the Discord discussion: typically the intent behind var-editing a fixed-transfer-rate container's transfer rate would be that it should now be stuck at the new value, and not let the player just revert it, no?

Reply:
Unless something is created to be var edited or needs special protections, things shouldn't be coded with admins var editing it in mind persay, especially if it's adding code complexity only for that. This was a discussion that was had in the syringe gun rework PR aswell.